docs/man/upscli_connect.txt

```diff
@@ -24,7 +24,7 @@ connection, or `UPSCLI_CONN_REQSSL` to require a SSL connection.
 
 Introduced in version 2.7, an additional flag `UPSCLI_CONN_CERTVERIF`
 now exists to verify the signature offered during the SSL handshake.
-This flag should be used in conjuction with linkman:upscli_init[3]
+This flag should be used in conjunction with linkman:upscli_init[3]
```

Comment:
I even had that little bit of indignation that a lowly spell check broke my build, and how dare it? I was obviously just going to have to define some new words for it to pass but then, sure enough, an *actual* typo! Yikes. Shame on me. :-P